ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -348,4 +348,17 @@ platform:
     preserveBootstrapIgnition: true
 EOF
   yq-go m -a -x -i "${CONFIG}" "${patch_bootstrap_ignition}"
-fi
\ No newline at end of file
+fi
+
+
+if [[ ${COMPUTE_NODE_VOLUME_TYPE} != "" ]]; then
```

Comment:
How about introduce a dedicated step to set volume type, something similar with `ipi-conf-azure-disktype`

Reply:
Added `ipi-conf-aws-rootvolume`